org.eclipse.xtext.xtext.wizard/build.gradle

```diff
@@ -3,11 +3,18 @@ description = 'Platform-independent component for creating new Xtext projects.'
 
 dependencies {
 	compile project(':org.eclipse.xtext')
-	optional "args4j:args4j:$versions.args4j"
+	compile "args4j:args4j:$versions.args4j"
```

Comment:
Am not sure if we want this

Reply:
It won't build an executable application otherwise.